backends/pkg/reconciler/eventmesh/handler.go

```diff
@@ -118,3 +161,107 @@ func fetchEventTypes(eventTypeLister eventinglistersv1beta2.EventTypeLister, log
+		// if the trigger's broker is not set or if we haven't processed the broker, we can skip the trigger
+		if trigger.Spec.Broker == "" {
+			continue
+		}
+		brokerRef := NameAndNamespace(trigger.Namespace, trigger.Spec.Broker)
+		if _, ok := brokerMap[brokerRef]; !ok {
+			continue
+		}
+
+		// if the trigger has no subscriber, we can skip it, there's no relation to show on Backstage side
+		if trigger.Spec.Subscriber.Ref == nil {
+			continue
+		}
+
+		subscriberBackstageId, err := getSubscriberBackstageId(ctx, dynamicClient, trigger, logger)
+		if err != nil {
+			// do not stop the Backstage plugin from rendering the rest of the data, e.g. because
+			// there are no permissions to get a single subscriber resource
+			continue
+		}
+
+		// we only care about subscribers that are in Backstage
+		if len(subscriberBackstageId) == 0 {
+			continue
+		}
+
+		// build the list of event types that the subscriber is subscribed to
+		subscribedEventTypes := buildSubscribedEventTypes(trigger, brokerMap[brokerRef], etNameMap, logger)
+
+		// go over the event types and add the subscriber to the subscription map
+		for _, eventType := range subscribedEventTypes {
+			key := NameAndNamespace(trigger.Namespace, eventType)
+			if _, ok := subscriptionMap[key]; !ok {
+				subscriptionMap[key] = Subscription{
+					BackstageIds: make(map[string]struct{}),
+				}
+			}
+			subscriptionMap[key].BackstageIds[subscriberBackstageId] = struct{}{}
+		}
+
+	}
+
+	return &subscriptionMap, nil
+}
+
+func buildSubscribedEventTypes(trigger *eventingv1.Trigger, broker *Broker, etNameMap map[string]*EventType, logger *zap.SugaredLogger) []string {
+	// TODO: we don't handle the CESQL yet
+	if trigger.Spec.Filter != nil && len(trigger.Spec.Filter.Attributes) > 0 {
+		// check if "type" attribute is present
+		if subscribedEventType, ok := trigger.Spec.Filter.Attributes["type"]; ok {
+			// it can be present but empty
+			// in that case, we assume the trigger is subscribed to all event types
+			if subscribedEventType != eventingv1.TriggerAnyFilter {
+				// if type is present and not empty, that means the trigger is subscribed to a specific event type
+				return []string{subscribedEventType}
+			}
+		}
+	}
+
+	// if no filter or type is specified, we assume the resource is interested in all event types that the broker provides
+	subscribedEventTypes := make([]string, 0, len(broker.ProvidedEventTypes))
+	for _, eventType := range broker.ProvidedEventTypes {
+		if et, ok := etNameMap[eventType]; ok {
+			subscribedEventTypes = append(subscribedEventTypes, et.Type)
+		}
+	}
+
+	return subscribedEventTypes
+}
+
+func getSubscriberBackstageId(ctx context.Context, client dynamic.Interface, trigger *eventingv1.Trigger, logger *zap.SugaredLogger) (string, error) {
+	refGvr, _ := meta.UnsafeGuessKindToResource(schema.GroupVersionKind{
+		trigger.Spec.Subscriber.Ref.Group,
+		trigger.Spec.Subscriber.Ref.APIVersion,
+		trigger.Spec.Subscriber.Ref.Kind,
+	})
```

Comment:
Group is an experimental feature, we need to parse the `Group` and `Version` from `APIVersion`
Something like this:
```golang
meta.UnsafeGuessKindToResource(schema.FromAPIVersionAndKind(trigger.Spec.Subscriber.Ref.APIVersion, trigger.Spec.Subscriber.Ref.Kind))

Reply:
Done, thanks